src/extractors/CallActivityOutExtractor.js

```diff
@@ -0,0 +1,51 @@
+import { forEach, isArray } from 'min-dash';
+
+import inherits from 'inherits';
+
+import BaseExtractor from './BaseExtractor';
+
+import { getOutMappings } from '../util/ExtensionElementsUtil';
+
+import { createProcessVariable, addVariableToList } from '../util/ProcessVariablesUtil';
+
+
+export default function CallActivityOutExtractor() {
+  BaseExtractor.call(this);
+}
+
+CallActivityOutExtractor.extractVariables = function(options) {
```

Comment:
Here we are using `extractVariables` as a "class method", i.e. it is not bound to instance. Anyway, do we need object here?